File: Leetcode/PY/Leetcode171-Excel表序列号.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        n = len(columnTitle)
        ans = 0
        cnt = 1
        for i in range(n):
            ans = ans *  26 + (ord(columnTitle[i]) - ord('A') + 1)
        return ans

File: Leetcode/Rust/Leetcode171-Excel表序列号.rs
impl Solution {
    pub fn title_to_number(column_title: String) -> i32 {
        let (mut res,mut n) = (0,column_title.len());
        for i in 0..n{
            res = res * 26 + (column_title.as_bytes()[i] - b'A' + 1) as i32;
        }
        res
    }
}